client/src/utils/proxy.js

```diff
@@ -0,0 +1,139 @@
+export default class ProxyDB {
+  constructor(DB_URL) {
+    this.DB_URL = DB_URL;
+  }
+
+  async getData(path = null) {
+    const url = path ? `${this.DB_URL}/${path}` : this.DB_URL;
+    const method = 'GET';
+    let response;
+
+    await fetch(url)
+      .then((res) => {
+        if (!res.ok) {
+          throw Error(`error:${res.status},method:${method},url:${url}`);
+        }
+        return res.json();
+      })
+      .then((data) => {
+        console.log(`${method} Request done successfully`);
```

Comment:
remove this console.log
we dont put logs in our projects 
at least change it to `console.debug(...)`